packages/cli/src/lib/setup/setupInstall.ts

```diff
@@ -947,34 +988,30 @@ function Install(options) {
                 : new RegExp(`^system\\.adapter\\.${adapter}\\.\\d+$`);
 
         try {
-            const doc = await objects.getObjectView('system', 'instance', {
+            const doc = await this.objects.getObjectView('system', 'instance', {
                 startkey: `system.adapter.${adapter}${instance !== undefined ? `.${instance}` : ''}`,
                 endkey: `system.adapter.${adapter}${instance !== undefined ? `.${instance}` : ''}\u9999`
             });
```

Comment:
```suggestion
            const doc: {
                rows: ioBroker.GetObjectViewItem<ioBroker.InstanceObject>[];
            } = await this.objects.getObjectView('system', 'instance', {
                startkey: `system.adapter.${adapter}${instance !== undefined ? `.${instance}` : ''}`,
                endkey: `system.adapter.${adapter}${instance !== undefined ? `.${instance}` : ''}\u9999`
            });
```

Reply:
why adding the type explicitly if it is inferred correctly?
![Bildschirmfoto von 2022-06-14 11-15-42](https://user-images.githubusercontent.com/25484431/173541872-46e769d2-a288-49e6-ab55-d4baadd941a5.png)

Reply:
Huh, maybe I needed to recompile. It was not inferring the type for me.